src/Field/FileListField.php

```diff
@@ -36,14 +36,22 @@ class FileListField extends ListField
 	 * @return  array  The field option objects.
 	 *
 	 * @since   1.0
-	 * @todo    Add support for a translate_options element
 	 */
 	protected function getOptions()
 	{
 		$options = array();
 
-		// Inject the Text object into HtmlSelect
-		HtmlSelect::$text = $this->getText();
+		$select = new HtmlSelect;
+
+		// Try to inject the text object into the field
+		try
+		{
+			$select->setText($this->getText());
+		}
+		catch (\RuntimeException $exception)
+		{
+			// A Text object was not set, ignore the error and try to continue processing
```

Comment:
Not necessarily.  Here, we ignore the error since the intent is to make it possible to not require a Text object, so if all of the translate options are set to false then a Text object is an unnecessary dependency (if you notice, the rest of the uses are with that type of check in place).  Still need to work on the Select class to follow this logic.